SPECS/kernel-uvm/kernel-uvm.spec

```diff
@@ -0,0 +1,1109 @@
+%global security_hardening none
+%global debug_package %{nil}
+%define uname_r %{version}-%{release}
+
+%ifarch x86_64
+%define arch x86_64
+%define archdir x86
+%define config_source %{SOURCE1}
+%endif
+
+Summary:        Linux Kernel for Kata UVM
+Name:           kernel-uvm
+Version:        5.15.48.1
```

Comment:
Is the version of `kernel-uvm` supposed to match that of the `kernel`/`kernel-headers` packages? If so, please modify the contents of the `version_matching_groups` variable in `toolkit/scripts/check_entangled_specs.py` to reflect that, so we can use our PR checks to keep these packages in sync.

Reply:
Spoke with Chris offline. The kernel configs for kernel-uvm come from an outside source which is not in sync with the latest LTS version. This is similar to how kernel-rt is dependent on a patchset sync'd with a specific version of the kernel which may not be updated to LTS's version. Therefore, kernel-uvm and kernel-rt may be behind as we wait for those sources to become available and/or update the kernel to address CVEs.

Reply:
Sounds great, thanks Rachel!

Reply:
What Rachel wrote is true, especially for right now.
Depending on how things progress, we may need to deviate `kernel-uvm` from the kernel configs posted by the upstream kata releases. Meaning we may need to revisit syncing with the LTS version.